src/shared/components/palette/address/write-address-field.component.ts

```diff
@@ -74,8 +76,8 @@ export class WriteAddressFieldComponent extends AbstractFieldWriteComponent impl
   }
 
   focusAddressList() {
-    if (document.getElementById('addressList')) {
-      document.getElementById('addressList').focus();
+    if (document.getElementById(this.idFocus + '_addressList')) {
+      document.getElementById( this.idFocus + '_addressList').focus();
```

Comment:
why we need to introduce `this.idFocus`? 
Can't we just call `this.id()` here?

Reply:
`this.id()` was failing the unit tests due to some specific browser compatibility problems it seems, so this was the only solution I found,  any suggestions please welcome